includes/helpers/ManageWikiNamespaces.php

```diff
@@ -130,10 +130,12 @@ public function modify( int $id, array $data ) {
 	 * Remove a namespace
 	 * @param int $id Namespace ID
 	 * @param int $newNamespace Namespace ID to migrate to
+	 * @param bool $maintainPrefix|false
 	 */
-	public function remove( int $id, int $newNamespace ) {
+	public function remove( int $id, int $newNamespace, bool $maintainPrefix = false ) {
 		// Utilise changes differently in this case
 		$this->changes[$id] = [
+			'maintainPrefix' => $maintainPrefix,
```

Comment:
Changes is a standardised format, wouldn’t this be appropriate to be in ‘new’?

Reply:
fixed